specs/IsPinchZoomEnabled.md

```diff
@@ -0,0 +1,84 @@
+# Background
+
+There are two types of zoom in Chromium: Browser Zoom and Pinch-Zoom:
+- Browser zoom, referred to as “Page Zoom” or “Zoom” more generally, is what we get by using Ctrl + +(plus) or – (minus) or Ctrl + Mousewheel. This changes the size of a CSS pixel relative to a device independent pixel and so it will cause page layout to change. End developers can programmatically change browser Zoom properties including [IsZoomControlEnabled](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2settings?view=webview2-1.0.774.44#get_iszoomcontrolenabled) and [ZoomFactor](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2controller?view=webview2-1.0.774.44#get_zoomfactor). Setting ZoomFactor property causes the layout to change and enables scroll bar which lets users interact with the zoomed in content using mouse.
+- Pinch-zoom, referred to as “Page Scale” zoom, is performed as a post-rendering step, it changes the page scale factor property and scales the surface the web page is rendered onto when user perfoms a pinch zooming action. It does not change the layout but rather changes the viewport and clips the web content, the content outside of the viewport isn't visible onscreen and users can't reach this content using mouse.
+
+Currently, the first type of zoom control is supported in WebView2 and modifying it has no effect on page scale zoom. 
+
+In response to customer requests to be able to change the current functionality of page scale zoom in WebView2, the WebView2 team has introduced this Pinch Zoom API which allows end developers to disable or enable page scale zoom control via a setting.
+
+In this document we describe the new setting. We'd appreciate your feedback.
+
+
+# Description
+The default value for `IsPinchZoomEnabled` is `true`.
+When this setting is set to `false`, it disables the ability of the end users to use pinching motions on touch input enabled devices to scale the web content in the WebView2 and users cannot pinch zoom.
+
+
+# Examples
+```cpp
+wil::com_ptr<ICoreWebView2> webView;
+void SettingsComponent::TogglePinchZooomEnabled()
+{
+    wil::com_ptr<ICoreWebView2Settings> coreWebView2Settings;
+    // Get webView's current settings
+    CHECK_FAILURE(webView->get_Settings(&coreWebView2Settings));
+
+    BOOL enabled;
+    CHECK_FAILURE(coreWebView2Settings->get_IsPinchZoomEnabled(&enabled));
```

Comment:
It will be on ICoreWebView2Settings4 (or something like that). Should show a second wil::com_ptr<ICoreWebView2Settings4> that you cast to from the existing first coreWebView2Settings pointer.